Added docs for telegraf proxmox and vector (#6619)
VictoriaMetrics/VictoriaMetrics: docs/data-ingestion/Proxmox.md

---
title: Proxmox
weight: 1
sort: 1
menu:
  docs:
    identifier: "proxmox"
    parent: "data-ingestion"
    weight: 1

>     # sort: 1

Reviewer: should we remove this line and other similar lines in commited files?

Author: removed in all files

aliases:
  - /data-ingestion/proxmox.html

>   - /data-ingestion/Proxmox.html

Reviewer: @tenmozes can we make http router in Hugo to be case insensetive?

Reviewer 2: Hugo is case insensitve but GCS is not, so if we deploy our site somewhere else it will be case insensitive

Reviewer 2: the best thing that we can do is to use lowercase

---

> # Proxmox Data Ingestion

Reviewer: ```suggestion
```
this anchor will make the link look like `/data-ingestion/proxmox#proxmox-data-ingestion,` which is almost a palindrome. Shall we just rm this anchor?

> Since Proxmox Virtual Environment(PVE) and Proxmox Backup Server(PBS) support sending data using the InfluxDB We can use the InfluxDB write support built into VictoriaMetrics

Reviewer: ```suggestion
Since Proxmox Virtual Environment(PVE) and Proxmox Backup Server(PBS) support sending data using the InfluxDB We can use the InfluxDB write support built into VictoriaMetrics.
```
Would it make sense to add links to `Proxmox Virtual Environment` and `Proxmox Backup Server`?

Currently PVE and PBS only support using an Authorization Token for authentication and does not support basic auth or a username and password.

## Proxmox Virtual Environment (PVE)

> If want help Sending your data to Managed VictoriaMetrics check out [our blog](https://victoriametrics.com/blog/proxmox-monitoring-with-dbaas/).

Reviewer: ```suggestion
If you want to send data to Managed VictoriaMetrics check out blog post [Monitoring Proxmox VE via Managed VictoriaMetrics](https://victoriametrics.com/blog/proxmox-monitoring-with-dbaas/).
```

Author: I reworded it a different way please review

1. Login to PVE as an administrator
2. Go to DataCenter > MetricServer > Add > InfluxDB
3. Set the parameters as follows
  * Name: VictoriaMetrics (can be changed to any string)

>   * Server: the hostname or IP of your VictoriaMetrics Instance

Reviewer: does it require specifying the path? which VM port should user specify?

Author: you do not need to specify the influxdb write path but I haven't tested what happens if you need to modify the path for a multitenant setup.

the port is specified in a different form field in PVE but the url with the ip port is 1 form field in PBS I will add screenshots so this is more clear

>   * Port: This will vary depending on where you were you sending the data and your settings but here are the defaults for the various components

Reviewer: typo `where you were you`

Author: reworded please review

    - 8427 for vmauth

>     - 8429 for vmagent

Reviewer: I find it inconsistent to list all the ports here, and omitting them in the rest examples. 
Shall we list these ports once somewhere and refer to them in other docs? wdyt @tiny-pangolin @qphuongl ?

Author: I agree maybe we can add something in the `_index.md` that lists the default port for all the components?

Reviewer 3: I will note this down and resolve it later. The ports and general information of these components will be explained in the main document

    * 8429 for VMAgent
  * Protocol: use HTTPS if you have TLS/SSL configured otherwise use HTTPS

>   * Organization: leave empty since it doesn't get used

Reviewer 4: "Protocol: use HTTPS if you have TLS/SSL configured otherwise use `HTTPS`" -> Should be `HTTP`

  * Bucket: leave empty since it doesn't get used

>   * Token: Your Token From VMAuth or leave blank if you don't have authentication enabled

Reviewer: ```suggestion
  * Token: your token from VMAuth or leave blank if you don't have authentication enabled
```

  * If you need to ignore TLS/SSL errors check the advanced box and uncheck the verify certificate box

> 4. Run `` in VMUI or in the explore view in Grafana to verify are being received by VictoriaMetrics

Reviewer: missing value in ``. Typo in `to verify are being received`

> ## Proxmox Backup Server (PBS)

Reviewer: same comments as above
